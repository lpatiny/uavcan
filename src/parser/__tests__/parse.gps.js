'use strict';

const parse = require('../parse');
const kinds = require('../../kinds.json');

describe('uavtest gps', () => {
  it('one packet', () => {
    let kindGeo = kinds[1041];
    let data = [12, 14, 15];

    let result = parse(data, kindGeo);
    expect(result).toMatchSnapshot();
  });
});
