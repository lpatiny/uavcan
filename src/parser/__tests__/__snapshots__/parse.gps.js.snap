// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uavtest gps one packet 1`] = `
Object {
  "gimbalId": 12,
  "heightCm": 0,
  "heightReference": 0,
  "latitudeDeg1E7": 0,
  "longitudeDeg1E7": 15,
  "mode": Object {
    "commandMode": 14,
  },
}
`;
